plugins/tally-form/skills/tally-form/assets/example-lecture-consultation.md

```
@@ -0,0 +1,26 @@
+---
+options:
+  - 네, 관심 있어요
+  - 더 알아보고 싶어요
+  - 아직 모르겠어요
+theme: neutral
+---
+# 강의 상담 신청
+> 어떤 점이 궁금한지 들려주세요. 부담 없이 시작해요.
+
+### 신청자 소개
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)